src/lib/Runner.ts

```diff
@@ -46,6 +48,11 @@ export class Runner {
     setInterval(() => {
       this.next();
     }, 15 * 1000); // 15s
+
+    // call checkRunningLandRequests() function on a interval of 10 min to verify LAND_BUILD_TIMEOUT_TIME
+    setInterval(() => {
+      this.checkRunningLandRequests();
+    }, 10 * 60 * 1000);
```

Comment:
Should the interval of 10 min sufficient ? The scenario of hung system is quite rare. So, don't want to overload the system with too many checks.

Reply:
Yep I think 10 minutes is good